src/System.Private.Uri/tests/FunctionalTests/IriEncodingDecodingTests.cs

```diff
@@ -0,0 +1,154 @@
+        {
+            Uri uri;
+            string host = "http://a";
+            string path = "/%C3%88/";
+            for (int i = 0; i < RFC3986CompliantDecoding.GetLength(0); i++)
+            {
+                string sourceStr = host + path + "?" + RFC3986CompliantDecoding[i, 0];
+                uri = new Uri(sourceStr);
+                Assert.Equal(path + "?" + RFC3986CompliantDecoding[i, 1], uri.PathAndQuery);
+            }
+        }
+
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Requires fix shipping in .NET 4.7.2")]
+        public void Query_DangerousQuerySymbols_RFC3986CompliantQuery()
+        {
+            Uri uri;
+            string baseUri = "http://a/%C3%88/";
+            for (int i = 0; i < RFC3986CompliantDecoding.GetLength(0); i++)
+            {
+                string sourceStr = baseUri + "?" + RFC3986CompliantDecoding[i, 0];
+                uri = new Uri(sourceStr);
+                Assert.Equal("?" + RFC3986CompliantDecoding[i, 1], uri.Query);
+            }
+        }
+
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Requires fix shipping in .NET 4.7.2")]
+        public void Equality_DangerousQuerySymbols_RFC3986CompliantEquality()
+        {
+            string baseUri = "http://a/%C3%88/";
+            for (int i = 0; i < RFC3986CompliantDecoding.GetLength(0); i++)
+            {
+                Uri uriTest = new Uri(baseUri + "?" + RFC3986CompliantDecoding[i, 0]);
+                Uri uriTest1 = new Uri(baseUri + "?" + RFC3986CompliantDecoding[i, 1]);
+                Assert.Equal(uriTest, uriTest1);
+            }
+        }
+
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Requires fix shipping in .NET 4.7.2")]
+        public void Fragment_DangerousFragmentSymbols_RFC3986CompliantFragment()
+        {
+            Uri uri;
+            string baseUri = "http://a/%C3%88/";
+            for (int i = 0; i < RFC3986CompliantDecoding.GetLength(0); i++)
+            {
+                string sourceStr = baseUri + "#" + RFC3986CompliantDecoding[i, 0];
+                uri = new Uri(sourceStr);
+                Assert.Equal("#" + RFC3986CompliantDecoding[i, 1], uri.Fragment);
+            }
+        }
+
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Requires fix shipping in .NET 4.7.2")]
+        public void UserInfo_DangerousUserInfoSymbols_RFC3986CompliantUserInfo()
+        {
+            Uri uri;
+            string baseUri = "http://";
+            string host = "a";
+            string path = "/%C3%88/";
+            for (int i = 0; i < RFC3986CompliantDecoding.GetLength(0); i++)
+            {
+                string sourceStr = baseUri + RFC3986CompliantDecoding[i, 0] + "@" + host + path;
+                uri = new Uri(sourceStr);
+                Assert.Equal(RFC3986CompliantDecoding[i, 1], uri.UserInfo);
+            }
+    }
+    }
+}
```

Comment:
nit: add a newline

Reply:
Should be fixed now, as well as the incorrect indentation there on line 152.